src/Components/Home.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Delivery from './Delivery'
+import DiningOut from './DiningOut'
+import Options from './Options'
+import { useState } from 'react'
+
+const myContext=React.createContext();
+export default function Home() {
+    const tog = false;
+    const [toggle, setToggle] = useState({ tog });
+    console.log(toggle);
+    return (
+        <div>
+            <myContext.Provider value={[toggle,setToggle]}>
```

Comment:
revisit this as per our discussion during meet. Show these each option as a submenu- use antd here. clicking on delivery should take to /delivery page and similarly for dining out. Also rename option1 and option2 accordingly.